features/login.feature

```diff
@@ -0,0 +1,38 @@
+Feature: user login 
+  
+    As a 61a TA
+    So that I can login to the hintr app
+    I want to be able to login and see the dashboard
+    
+Background: user already has an account
+  
+  Given the following accounts exist:
+    | email                     | password          | admin |
+    | testuser@gmail.com        | password          | 0     |
+  
+Scenario: Not logged in user will get redirected to login page
+  Given I am not logged in
+  Given I visit the home page
+  Then I should see the login page
```

Comment:
Page checks should say "Then I should BE ON the ______ page" instead of "see".  See can be used for view checks.